dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/IdentifiableObjectManager.java

```diff
@@ -67,7 +67,10 @@
 
     <T extends IdentifiableObject> T get( Class<T> type, long id );
 
-    <T extends IdentifiableObject> T get( Class<T> type, String uid );
+    <T extends IdentifiableObject> T get( Class<T> type, String uid )
```

Comment:
Just as food for thought: If we have a `get` that makes sure things exist a lookup that allows for things to not exist is usually called `find` or something like that. We might want to look into establishing semantics consistent with method names used.

Reply:
Yes I agree, it would be good with a more succinct notation for null vs exception. Could be good to do a separate refactor PR.